_posts/2018/2018-03-04-only-use-named-scopes-ouside-models.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Only use named scopes outside models'
+description: 'Keep your wheres to yourself'
+layout: article
+category: ruby
+image:
+  base: '2018/only-use-named-scopes-outside-models'
+  alt: 'San Francisco Museum of Modern Art'
+  source: 'https://unsplash.com/photos/RkJF2BMrLJc'
+  credit: 'William Bout'
+---
+
+Last time we discussed using the hash-syntax in your `#where` methods but in my examples I did something I wouldn't do in my real-life code...
```

Comment:
*'hash syntax' or 'hash-style syntax' (the latter for continuity, but not important)